react/tic-tac-toe/src/app/screens/Home/components/Navbar/styles.module.scss

```diff
@@ -0,0 +1,61 @@
+
+@import 'src/scss/colors';
+@import 'src/scss/variables';
+
+.navbar {
+  pointer-events: auto;
+  position: relative;
+  top: 0;
+}
+.container {
+  align-items: center;
+  background: $white;
+  box-shadow: rgba(0, 0, 0, 0.06) 0 6px 20px;
```

Comment:
Does this work? I'm quite sure the color of the shadow goes at the end of the definition.

Reply:
I don't know, but it works  🙄

Reply:
Maybe it's SASS